test/lightning_web/live/workflow_live/index_test.exs

```diff
@@ -17,6 +18,34 @@ defmodule LightningWeb.WorkflowLive.IndexTest do
              |> element("#workflows-#{project.id}", "No workflows yet")
     end
 
+    test "only users with MFA enabled can access workflows for a project with MFA requirement",
+         %{
+           conn: conn
+         } do
+      user = user_with_mfa_fixture()
+      conn = log_in_user(conn, user)
+
+      project =
+        insert(:project,
+          requires_mfa: true,
+          project_users: [%{user: user, role: :admin}]
+        )
```

Comment:
I've changed one or two of the previous calls to `Projects.create_project(...)`, as with the request about the mfa fixture, please when writing new code could you try and not use things for the fixtures module and rather use/extend the `Factories` module.